### ISSUE TITLE
Fix shadow

### ISSUE THREAD

**Issue by krharsh17** [krharsh17/barview-android#91]
**Describe the bug**
The shadow's controls are completely broken. Either fix their XML attributes and provide proper documentation in the README.md for them or remove the shadows entirely.

**Comment by spacemonkey1101** [comment]
would like to work on this.

**Comment by spacemonkey1101** [comment]
Could I change the these files :
Bar.java
BarView.java
BarGroup.java

**Comment by sidhuparas** [comment]
@spacemonkey1101 I think you can modify any suitable file.

**Comment by sidhuparas** [comment]
@spacemonkey1101 Any update?

**Comment by spacemonkey1101** [comment]
im actually struggling with it. Maybe im not getting it

**Comment by sidhuparas** [comment]
@spacemonkey1101 Are you having issue understanding the problem? Or are you not getting how to solve it or update the documentation? Me and @krharsh17 are here to help you out.

**Comment by spacemonkey1101** [comment]
yes Im facing an issue...... I tried to modify some files and values to get the bars to the ideal config.....but i failed repeatedly

**Comment by krharsh17** [comment]
@rajkothari634 has worked on the shadow, maybe he can help in removing/fixing it?
